apps/mark/frontend/src/pages/print_page.tsx

```diff
@@ -16,12 +17,27 @@ export function PrintPage(): JSX.Element {
     resetBallot,
   } = useContext(BallotContext);
   assert(electionDefinition, 'electionDefinition is not defined');
-  assert(typeof ballotStyleId === 'string', 'ballotStyleId is not defined');
-  assert(typeof precinctId === 'string', 'precinctId is not defined');
 
   const printerTimer = useRef(0);
 
-  function onPrintStarted() {
+  async function print() {
```

Comment:
had to move this from `mark-flow-ui` to here until it is moved to the backend in the next PR